core/domain/question_domain.py

```diff
@@ -308,6 +309,27 @@ def _convert_state_v32_dict_to_v33_dict(cls, question_state_dict):
 
         return question_state_dict
 
+    @classmethod
+    def _convert_state_v33_dict_to_v34_dict(cls, question_state_dict):
+        """Converts from version 33 to 34. Version 34 adds a new
+        attribute math RTEs. The new attribute has an additional field to
+        for storing SVG filenames.
+
+        Args:
+            question_state_dict: dict. A dict where each key-value pair
+                represents respectively, a state name and a dict used to
+                initialize a State domain object.
+
+        Returns:
+            dict. The converted question_state_dict.
+        """
+        question_state_dict = (
+            state_domain.State.convert_html_fields_in_state(
```

Comment:
Ditto (can we move this to the line above? same for the add_math_content_to_math_rte_components

Reply:
done